addons/l10n_in/models/account_invoice.py

```diff
@@ -49,7 +50,7 @@ def _compute_l10n_in_gst_treatment(self):
     @api.depends('partner_id', 'partner_shipping_id', 'company_id')
     def _compute_l10n_in_state_id(self):
         for move in self:
-            if move.country_code == 'IN' and move.journal_id.type == 'sale':
+            if move.country_code == 'IN' and move.is_invoice(include_receipts=True):
```

Comment:
```suggestion
            if move.country_code == 'IN' and move.is_sale_document(include_receipts=True):
```

Reply:
We need this to be applied on both sales and purchase documents :slightly_smiling_face:.

Reply:
@masm-odoo not right now.
so you can remove it.